units/unexpirable_spawnhost_stats.go

```diff
@@ -69,12 +69,27 @@ func (j *unexpirableSpawnHostStatsJob) Run(ctx context.Context) {
 	stats := j.getStats(hosts)
 
 	grip.Info(message.Fields{
-		"message":                      "unexpirable spawn host stats",
-		"job_id":                       j.ID(),
-		"total_uptime_secs":            stats.totalUptime.Seconds(),
-		"uptime_secs_by_distro":        stats.uptimeSecsByDistro,
-		"uptime_secs_by_instance_type": stats.uptimeSecsByInstanceType,
+		"message":           "unexpirable spawn host stats",
+		"job_id":            j.ID(),
+		"total_uptime_secs": stats.totalUptime.Seconds(),
 	})
+
+	for distroID, uptimeSecs := range stats.uptimeSecsByDistro {
```

Comment:
Not really for this PR, but more general question: I think this would do better in honeycomb. Should we create a ticket to eventually switch it over?

Reply:
We can't use Honeycomb because it samples the results and samples Amboy jobs (unless you do something awkward like set an error on the trace). Since we already have other monitoring for sleep schedules in Splunk and would have to write a weird workaround to maneuver around the sampling rules, it seems like Splunk is just easier.